### ISSUE TITLE
Experiments: Test hyperparameters

### ISSUE THREAD

**Issue by mkduer** [mkduer/reinforcement-learning-snake-game#34]
- [x] learning rate #64 -- optimal rate 0.005
- [x] epsilon factor #75  -- optimal rate 0.1
- [x] discount factor #74 -- optimal rates 0.85-0.95
- [x] rewards #73 

DEFAULT VALUES for all tests (excepting the one being tested):
Learning Rate: 0.005
Epsilon: 0.1
Discount Factor: 0.9
Rewards: Wall -100, Body -100, Empty -10, Mouse 100
